be/src/vec/exec/format/orc/vorc_reader.h

```diff
@@ -90,7 +90,7 @@ struct OrcPredicate {
 };
 
 struct LazyReadContext {
-    VExprContext* vconjunct_ctx = nullptr;
+    VExprContextSPtrs* conjuncts = nullptr;
```

Comment:
VExprContextSPtrs is ok, not need VExprContextSPtrs*?

Reply:
It needs`VExprContextSPtrs*`.